packages/app/cypress/e2e/runner/reporter.errors.cy.ts

```diff
@@ -87,4 +86,908 @@ describe('errors ui', {
+      })
+    })
+
+    verify.it('onAbort assertion failure', {
+      file,
+      column: 29,
+      codeFrameText: 'onAbort',
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('onAbort exception', {
+      file,
+      column: 14,
+      codeFrameText: 'onAbort',
+      message: 'bar is not a function',
+    })
+
+    verify.it('onRequest assertion failure', {
+      file,
+      column: 29,
+      codeFrameText: 'onRequest',
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('onRequest exception', {
+      file,
+      column: 14,
+      codeFrameText: 'onRequest',
+      message: 'bar is not a function',
+    })
+
+    verify.it('onResponse assertion failure', {
+      file,
+      column: 29,
+      codeFrameText: 'onResponse',
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('onResponse exception', {
+      file,
+      column: 14,
+      codeFrameText: 'onResponse',
+      message: 'bar is not a function',
+    })
+  })
+
+  describe('cy.readFile', () => {
+    const file = 'readfile.cy.js'
+
+    before(() => {
+      setup({
+        fileName: file,
+        mockPreferredEditor: false,
+        onLoadStatsMessage: 'Failed:1',
+      })
+    })
+
+    verify.it('existence failure', {
+      file,
+      column: 8,
+      message: 'failed because the file does not exist',
+    })
+  })
+
+  describe('validation errors', () => {
+    const file = 'validation.cy.js'
+
+    before(() => {
+      setup({
+        fileName: file,
+        mockPreferredEditor: false,
+        onLoadStatsMessage: 'Failed:3',
+      })
+    })
+
+    verify.it('from cypress', {
+      file,
+      column: 8,
+      message: 'can only accept a string preset or',
+      stack: ['throwErrBadArgs', 'From Your Spec Code:'],
+    })
+
+    verify.it('from chai expect', {
+      file,
+      column: '(5|12)', // (chrome|firefox)
+      message: 'Invalid Chai property: nope',
+      stack: ['proxyGetter', 'From Your Spec Code:'],
+    })
+
+    verify.it('from chai assert', {
+      file,
+      column: 12,
+      message: 'object tested must be an array',
+    })
+  })
+
+  describe('event handlers', () => {
+    const file = 'events.cy.js'
+
+    before(() => {
+      setup({
+        fileName: file,
+        mockPreferredEditor: false,
+        onLoadStatsMessage: 'Failed:3',
+      })
+    })
+
+    verify.it('event assertion failure', {
+      file,
+      column: 27,
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('event exception', {
+      file,
+      column: 12,
+      message: 'bar is not a function',
+    })
+
+    verify.it('fail handler assertion failure', {
+      file,
+      column: 27,
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('fail handler exception', {
+      file,
+      column: 12,
+      message: 'bar is not a function',
+    })
+  })
+
+  describe('uncaught errors', () => {
```

Comment:
Nit: i know these were pulled forward, but it'd be easier to reason with if these were grouped into sync & async context's 
```
describe('uncaught errors', () => {
  describe('sync', () => {})
  describe('async', () => {})

``